### PR TITLE
change all uses of Stokes to pseudo-Stokes

### PR DESCRIPTION
**Pull request by bhazelton**

### CODE REVIEW

File: pyuvdata/beamfits.py [hunk 1]
```diff
@@ -128,8 +128,13 @@ def read_beamfits(self, filename, run_check=True, check_extra=True,
                 self.data_array = np.squeeze(complex_arrs[0] + 1j * complex_arrs[1], axis=0)
             else:
                 self.data_array = data
+
+            # Note: This axis is called STOKES by analogy with the equivilent uvfits axis
+            # However, this confusing because it is NOT a true Stokes axis,
```
Comment (adampbeardsley) [inline pyuvdata/beamfits.py]: Couple typos: equivilent -> equivalent; "this confusing" -> "this is confusing"

File: pyuvdata/beamfits.py [hunk 2]
```diff
@@ -449,8 +454,11 @@ def write_beamfits(self, filename, run_check=True, check_extra=True,
             else:
                 pol_spacing = 1
 
+            # Note: This axis is called STOKES by analogy with the equivilent uvfits axis
+            # However, this confusing because it is NOT a true Stokes axis,
```
Comment (adampbeardsley) [inline pyuvdata/beamfits.py]: Same typos

File: pyuvdata/beamfits.py (hunk 1 above)
Comment (bhazelton) [reply pyuvdata/beamfits.py]: thanks, fixing them now

File: pyuvdata/beamfits.py (hunk 2 above)
Comment (bhazelton) [reply pyuvdata/beamfits.py]: thanks, fixing them now

**Comment by bhazelton** [comment]
I'm a little worried about the downstream affect on users, but I think that these are only recently being used, so maybe we can make this change quickly and not affect too many people? That's part of why I asked @nkern to look at this too. I'm just very reluctant to be a source of more confusion about this going forward...

**Comment by nkern** [comment]
I think a good compromise would be to have `polnum2str` map from `1 -> pI`, but support compatibility in `polstr2num` (as Adam suggested) to do both `pI -> 1` and `I -> 1`, to support users who have adopted that convention.

**Comment by bhazelton** [comment]
Ok , that sounds reasonable @nkern, I'll make that change

File: pyuvdata/utils.py [hunk 3]
```diff
@@ -315,14 +315,15 @@ def fits_indexhdus(hdulist):
...
+    Use 'pI', 'pQ', 'pU' and 'pV' to make it clear that these are pseudo-Stokes, not true Stokes
 
     Args:
         pol: polarization string
 
     Returns:
         Number corresponding to string
     """
-    poldict = {'I': 1, 'Q': 2, 'U': 3, 'V': 4,
+    poldict = {'PI': 1, 'PQ': 2, 'PU': 3, 'PV': 4,
```
Comment (nkern) [inline pyuvdata/utils.py]: this is supposed to be lower case on the `p`, no?

Comment (nkern) [reply pyuvdata/utils.py]: ah, is this b/c you do `pol.upper()` before passing through to the dictionary?

Comment (bhazelton) [reply pyuvdata/utils.py]: Yes, but the way the `polstr2num` function handles case insensitivity is to make them all upper case. I decided case-insensitivity was a good thing for `polstr2num`, but I used lowercase `p` for `polnum2str`.

Comment (nkern) [reply pyuvdata/utils.py]: ok makes sense. i'm fine w/ this then.

Comment (bhazelton) [reply pyuvdata/utils.py]: I can add a comment to that effect...

**Pull request merged**